### ISSUE TITLE
Trust Token API

### ISSUE THREAD

**Issue by dvorak42** [antifraudcg/proposals#7]
Following up from the presentation today, I'd like to eventually propose moving the Trust Token API work from the WICG to the Anti-Fraud CG.

Current Documents: https://github.com/WICG/trust-token-api/

I'm opening this issue for early feedback about moving the API, prior to sending out a call to adopt the work on the list.

Please provide feedback/thoughts on this issue or via the Slack.

**Comment by SpaceGnome** [comment]
What are the pros/cons from moving this from WICG to the Anti-Fraud CG?

**Comment by dvorak42** [comment]
Mostly just pros. The Anti-Fraud CG has regular meetings and can provide more feedback and direction as we clean up the API on the path to standardization and the API seems generally useful for the Anti-Fraud space.

**Comment by chris-wood** [comment]
If Trust Token is going to move, I think it needs to be updated to align with the latest Privacy Pass architecture and protocol.

**Comment by dvorak42** [comment]
Yeah, do you think that needs to happen before it is adopted by the CG, or can it be done within the CG?

From the slides, the rough plan is:
- Proposal to move Trust Token API (WICG)
- Align with PrivacyPass protocol changes.
- Build up strategies for using tokens in Anti-Fraud.
- Corresponding proposals to establish the initial fraud signal.
- Move to WG (WebAppSec?)

**Comment by chris-wood** [comment]
I think I'd prefer that happen before it's adopted by the CG, specifically because I think the CG should be focusing on defining the requirements for signals. The mechanism for presenting the signal (Trust Token built on Privacy Pass) is less interesting (to me).

**Comment by claucece** [comment]
That seems like a wonderful idea! We didn't have much time last meeting to discuss the proposals, so perhaps we can bring this up on the next one.

**Comment by dvorak42** [comment]
We'll be briefly (3-5 minutes) going through open proposals at the Anti-Fraud CG meeting this week. If you have a short 2-4 sentence summary/slide you'd like the chairs to use when representing the proposal, please attach it to this issue otherwise the chairs will give a brief overview based on the initial post.

**Comment by dvorak42** [comment]
Potentially of interest to folks who were looking at/providing feedback to this API, the Google Chrome team will be hosting office hours on it to help answer questions about the current state of the API in Chrome and resolve any gaps in understanding between the current version/documentation/use cases. Hopefully we'll be able to improve documentation and find directions where future extensions or related proposals can be extended/built.

The North America/LATAM/EMEA/APAC friendly session is happening on Wednesday March 6th at 8am-9am PST || 11am-12pm EST || 4pm-5pm GMT. A Portuguese language session is happening February 27 at 10-11 am PST.  

[More details and registration is available here.](https://rsvp.withgoogle.com/events/privacy-sandbox-developer-office-hours-private-state-tokens-en)

If there are particular learnings/FAQs from the session that are relevant to the AFCG, we'll post them here and if there's interest ask for some time at a future AFCG meeting.
